src/scripts/datadog-docs.js

```diff
@@ -1,3 +1,4 @@
+import Cookies from 'js-cookie';
```

Comment:
Is this import statement needed? I'm not seeing it referenced anywhere else in this file

Reply:
Yea you are right it was from previous code changes, removed now